Tools/CineMachine-Recipes/CineMachineBuilder.cs

```diff
@@ -0,0 +1,13 @@
+using RecipeEngine.Api.Jobs;
+using RecipeEngine.Platforms;
+
+namespace Cinemachine.Cookbook;
+
+public class CineMachineBuilder
```

Comment:
So normally Cinemachine is spelled Cinemachine, not CineMachine.  Don't know if that's an issue for you, but consistency would be nice.  That would apply to the folder and file name too.

Reply:
Will do.